storage/src/vespa/storage/distributor/distributor.cpp

```diff
@@ -558,7 +584,12 @@ Distributor::propagateInternalScanMetricsToExternal()
     if (_use_legacy_mode) {
         _stripe->propagateInternalScanMetricsToExternal();
     } else {
-        first_stripe().propagateInternalScanMetricsToExternal();
+        _metrics_sum->reset();
```

Comment:
@vekterli should comment on how this interacts with __metrics_sum_ also being the metric set that is registered with the component (ref L120).

Reply:
Resetting here (i.e. effectively recursively zeroing all child metrics) for `_metrics_sum` should be fine since this function is called in the context of a metric update callback, so there shouldn't be any concurrent meddling from the metric manager.

However, the biggest head-scratcher is how the underlying stripe metrics will interact with periodic metric snapshotting, since I believe it needs to do implicit zeroing of registered metrics to handle deltas. We might need an offline discussion on this. 🎷🦖